test/loading.jl

```diff
@@ -14,6 +14,9 @@ include_string_test_func = include_string("include_string_test() = @__FILE__", t
 
 @test isdir(@__DIR__)
 @test @__DIR__() == dirname(@__FILE__)
+let exename = `$(Base.julia_cmd()) --precompiled=yes --startup-file=no`
+    @test readchomp(`$exename -E "@__DIR__" -i`) == string('"', pwd(), '"')
+end
```

Comment:
Test that you don't get pwd() when running in a script? E.g. by doing `cd("/tmp") do @__DIR__; end`?